<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plot_navigator</class>
 <widget class="QFrame" name="plot_navigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="CustomToolButton" name="origin_view_button">
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="icon/plot_navigator.qrc">
       <normaloff>:/navigator/home-60.png</normaloff>:/navigator/home-60.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CustomToolButton" name="vertical_marker_button">
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="icon/plot_navigator.qrc">
       <normaloff>:/navigator/vertical-line-60.png</normaloff>:/navigator/vertical-line-60.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="after_view_button">
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="icon/plot_navigator.qrc">
       <normaloff>:/navigator/ruler-60.png</normaloff>:/navigator/ruler-60.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pan_view_button">
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="icon/plot_navigator.qrc">
       <normaloff>:/navigator/drag-60.png</normaloff>:/navigator/drag-60.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zoom_button">
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="icon/plot_navigator.qrc">
       <normaloff>:/navigator/search-60.png</normaloff>:/navigator/search-60.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setting__button">
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="icon/plot_navigator.qrc">
       <normaloff>:/navigator/adjust-60.png</normaloff>:/navigator/adjust-60.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save_button">
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="icon/plot_navigator.qrc">
       <normaloff>:/navigator/save-60.png</normaloff>:/navigator/save-60.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="position_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="msg_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomToolButton</class>
   <extends>QToolButton</extends>
   <header>.custom_tool_button</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon/plot_navigator.qrc"/>
 </resources>
 <connections/>
</ui>
